python/paddle/fluid/tests/unittests/dygraph_to_static/test_word2vec.py

```diff
@@ -0,0 +1,325 @@
+               negative_sample_num=10):
+
+    dataset = []
+
+    for line in corpus:
+        for center_word_idx in range(len(line)):
+            window_size = random.randint(1, max_window_size)
+            center_word = line[center_word_idx]
+
+            positive_word_range = (max(0, center_word_idx - window_size), min(
+                len(line) - 1, center_word_idx + window_size))
+            positive_word_candidates = [
+                line[idx]
+                for idx in range(positive_word_range[0], positive_word_range[1]
+                                 + 1)
+                if idx != center_word_idx and line[idx] != line[center_word_idx]
+            ]
+
+            if not positive_word_candidates:
+                continue
+
+            for positive_word in positive_word_candidates:
+                dataset.append((center_word, positive_word, 1))
+
+            i = 0
+            while i < negative_sample_num:
+                negative_word_candidate = random.randint(0, vocab_size - 1)
+
+                if negative_word_candidate not in positive_word_candidates:
+                    dataset.append((center_word, negative_word_candidate, 0))
+                    i += 1
+
+    return dataset
+
+
+dataset = build_data(corpus, word2id_dict, word2id_freq)
+for _, (center_word, target_word, label) in zip(range(50), dataset):
+    print("center_word %s, target %s, label %d" %
+          (id2word_dict[center_word], id2word_dict[target_word], label))
+
+
+def build_batch(dataset, batch_size, epoch_num):
+
+    center_word_batch = []
+    target_word_batch = []
+    label_batch = []
+    eval_word_batch = []
+
+    for epoch in range(epoch_num):
+
+        #random.shuffle(dataset)
+
+        for center_word, target_word, label in dataset:
+            center_word_batch.append([center_word])
+            target_word_batch.append([target_word])
+            label_batch.append([label])
+
+            if len(eval_word_batch) < 5:
+                eval_word_batch.append([random.randint(0, 99)])
+            elif len(eval_word_batch) < 10:
+                eval_word_batch.append([random.randint(0, vocab_size - 1)])
+
+            if len(center_word_batch) == batch_size:
+                yield np.array(center_word_batch).astype("int64"), np.array(
+                    target_word_batch).astype("int64"), np.array(
+                        label_batch).astype("float32"), np.array(
+                            eval_word_batch).astype("int64")
+                center_word_batch = []
+                target_word_batch = []
+                label_batch = []
+                eval_word_batch = []
+
+    if len(center_word_batch) > 0:
+        yield np.array(center_word_batch).astype("int64"), np.array(
+            target_word_batch).astype("int64"), np.array(label_batch).astype(
+                "float32"), np.array(eval_word_batch).astype("int64")
+
+
+#for _, batch in zip(range(10), build_batch(dataset, 128, 3)):
+#    print(batch)
```

Comment:
The same as above

Reply:
The code is at the original code. I commented it so that we don't print much info. I will remove line 208-209.